commons/src/main/java/io/aiven/kafka/tiered/storage/commons/chunkindex/AbstractChunkIndexBuilder.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2023 Aiven Oy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.aiven.kafka.tiered.storage.commons.chunkindex;
+
+public abstract class AbstractChunkIndexBuilder implements ChunkIndexBuilder {
+    protected final int originalChunkSize;
+    protected final int originalFileSize;
+
+    protected int chunksAdded = 0;
```

Comment:
Does this need to be protected? It's only used in this class.

Reply:
Nah, same for `finished`. Will make `private`.